src/v/debug_bundle/debug_bundle_service.cc

```diff
@@ -99,6 +106,31 @@ std::filesystem::path form_debug_bundle_storage_directory() {
       config::node().data_directory.value().path
       / service::debug_bundle_dir_name);
 }
+
+ss::future<bytes> calculate_sha256_sum(std::string_view path) {
+    const size_t buffer_size = 8192;
```

Comment:
Seems kinda small, but I don't know how to tune it, perhaps on how long it takes to sha256 a chunk?

Can the read_ahead be constantified here, too, please.

Reply:
> Seems kinda small, but I don't know how to tune it, perhaps on how long it takes to sha256 a chunk?

Yeah probably way too small.  I actually was going with the size of our fragmented vector to try to reduce memory fragmentation.  Main reason to break this up is in case the file is large (some bundles I've heard get into the 100's of MB) and I didn't want to read it all into memory at once.  Our limit of oversized alloc is 128KiB, so maybe I"ll bump this to half that